Applications/VGG/jni/meson.build

```diff
@@ -1,8 +1,10 @@
 build_root = meson.build_root()
 res_path = meson.current_source_dir() / '..' / 'res'
+cifar_path = meson.current_source_dir() / '../../utils/datagen/cifar'
```

Comment:
Please make meson.build file in utils/datagen/cifar and declare cifar_path in there.

Reply:
Fix